aws/resource_aws_api_gateway_integration.go

```diff
@@ -55,6 +55,21 @@ func resourceAwsApiGatewayIntegration() *schema.Resource {
 				}, false),
 			},
 
+			"connection_type": {
```

Comment:
We're missing `d.Set("connection_type", integration.ConnectionType)` in the read function 😄

Reply:
what a shame 😱